pkgs/development/compilers/rust/cargo_cross.nix

```diff
@@ -0,0 +1,23 @@
+
+{stdenv, pkgsBuildBuild, makeShellWrapper, cacert, rustc}:
+
+stdenv.mkDerivation {
```

Comment:
`runCommand` is probably more suitable for this.  Should also inherit most of `meta`, and link to all the non-executable stuff like man pages, because people will want to use the cross cargo in nix shells.

Reply:
I've updated to runCommand, and inherited meta